Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:experimental
 
-FROM quay.io/unstructured-io/base-images:centos7.9
+FROM quay.io/unstructured-io/base-images:centos7.9-1
```

Comment:
Does this correspond to a change in [this repo](https://github.com/Unstructured-IO/base-images)? I would think we don't need a `install-pandoc` target in here if the installation happens in the base image.

Reply:
the centos 7.9-1 is supposed to be built once this https://github.com/Unstructured-IO/base-images/pull/1 is merged by ci, but it might fail because of how long it takes on the slow GHA runners (and i believe Trevor will have a solution later). And currently, the new image is built on large ec2 instance manually. you can find it at:https://quay.io/repository/unstructured-io/base-images?tab=tags&tag=centos7.9

Reply:
the `install-pandoc` is for `test-unit` on CI Ubuntu system, the command in ci before is also out-dated:https://quay.io/repository/unstructured-io/base-images?tab=tags&tag=centos7.9

Reply:
Ah, makes sense!

Reply:
i know it sounds repeating the installation here but since the pandoc is installed in base-image, i could't share it here with ci